buildSrc/src/main/java/Dependencies.kt

```diff
@@ -31,7 +31,7 @@ object Versions {
 
     const val mozilla_appservices = "82.1.0"
 
-    const val mozilla_glean = "39.0.3"
+    const val mozilla_glean = "40.0.0"
```

Comment:
Do we need this? I thought the intention was that AC wouldn't depend on glean directly anymore

Reply:
Unfortunately yes. I extended the commit message, also visible in the PR description above now, with some reasoning.

Note: we explicitly called this out in the doc back then as an optional step to try. I will file a followup bug to figure out how we can improve that.